tracpro/orgs_ext/middleware.py

```diff
@@ -0,0 +1,32 @@
+from django.utils.translation import ugettext_lazy as _
+from django.http import HttpResponseBadRequest
+
+from temba_client.base import TembaAPIError, TembaConnectionError
+
+
+class HandleTembaAPIError(object):
+    """ Catch all Temba exception errors """
+
+    def process_exception(self, request, exception):
+
+        rapidProConnectionErrorString = _(
```

Comment:
By [convention](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables), Python variable names are written in lower case with underscores replacing spaces. :snake: